modules/SofaDenseSolver/SofaDenseSolverConfig.cmake.in

```diff
@@ -0,0 +1,13 @@
+# CMake package configuration file for the SofaDenseSolver module
+@PACKAGE_INIT@
+
+sofa_find_package(Newmat REQUIRED)
+sofa_find_package(SofaBase REQUIRED)
+
```

Comment:
Do not use `sofa_find_package` in Config.cmake, it needs SofaMacros.cmake and it's not sure the caller already has it.